server/http.go

```diff
@@ -39,6 +39,7 @@ const (
 	routeIncomingWebhook           = "/webhook"
 	routeOAuth1Complete            = "/oauth1/complete.html"
 	routeOAuth1PublicKey           = "/oauth1/public_key.html" // TODO remove, debugging?
+	routeToDocsOrConnect           = "/user/docsorconn"
```

Comment:
For consistency among the routes. Although it looks like we already have a mixture of `-` and `_`. Most of them are `-` so let's go with that.

```suggestion
	routeToDocsOrConnect           = "/user/docs-or-connect"
```